sqlitex/exec.go

```diff
@@ -35,11 +35,12 @@ type ExecOptions struct {
 	//
 	// Basic reflection on Args is used to map:
 	//
-	//	integers to BindInt64
-	//	floats   to BindFloat
-	//	[]byte   to BindBytes
-	//	string   to BindText
-	//	bool     to BindBool
+	//	integers    to BindInt64
```

Comment:
Let's turn these into [lists](https://go.dev/doc/comment#lists) and remove the special spacing.

Reply:
For sure 👍  GitHub's UI editing swapped the original tabs for spaces - reasons not to trust it. I've opted to match the `sqlitex/pool.go` syntax for lists, the `sqlite.go` comment syntax has an extra space indent.